### PR TITLE
PD-8383 Updating our ROST page

### PR DESCRIPTION
**Pull request by ghost**
Adding in some awesome new tools from around Rapid7.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -79,14 +82,13 @@ <h3 style="color:#FF7F00">Take advantage of these tools to help secure your envi
...
+            <h2><i class="fa fa-cogs"></i> Configuration Management</h2>
+            <h3 style="color:#497CE2">Alter and secure your configuration without restarting or redeploying.</h3>
+            <p class="cat-desc">In this section, you'll find our easy-to-use centralized configuration management tools like <a href="https://github.com/rapid7/propsd">propsd</a> which combines the power of S3 along with <a href="https://www.consul.io/">Consul.io</a> to deliver dynamic configuration and catalog changes.  We are also including here our tooling for encrypting secrets via <a href="https://github.com/rapid7/tokend">tokend</a> and <a href="https://github.com/rapid7/warden">warden</a>.</p>
```
Comment (fmitchell-r7) [inline index.html]: Do we want to capitalize Propsd, Tokend, and Warden here? Also, I think it's just "Consul" and not "Consul.io".

**Review by todb-r7 — APPROVED** [review]
Look like fine additions to me. But, what's up with the leading empty tags, `<i class="fa fa-cogs"></i>`? I assume this this an artifact of page generation software.

**Comment by ghost** [comment]
@todb-r7 AFAIK it's how to properly add bootstrap icons to html (this terrible HTML was coded by your's truly from scratch, not generated).

See:
http://www.w3schools.com/icons/

**Comment by todb-r7** [comment]
Oh, of course, I didn't look at the rendered. Yes, it's weird, but Bootstrap is tons of fun regardless.

**Pull request merged**